WPMedia/fusion-news-theme-blocks
# PEN-1761 - Add new translations

## Description

Add new translations for de, es, ja, ko - with translation text

## Jira Ticket
- [PEN-1761](https://arcpublishing.atlassian.net/browse/PEN-1761)

## Acceptance Criteria

1. Spanish, German, Japanese and Korean are added as additional languages that can be supported in strings in Arc Blocks, based on the existing locale Theme Setting
2. Spanish, German, Japanese and Korean are added as additional languages that can be supported by date functionality, based on the existing Date localization language Theme Setting
3. Tests and internal documentation are updated to cover this new functionality

## Test Steps

1. Checkout this branch `git checkout PEN-1761-add-translations`
2. In Fusion repo checkout PR for locale changes ``
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/byline-block,@wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-promo-block,@wpmedia/full-author-bio-block,@wpmedia/results-list-block,@wpmedia/search-results-list-block,@wpmedia/header-nav-chain-block,@wpmedia/global-phrases-block`
4. Navigate to different pages using different sites to verify language strings

http://localhost/all-blocks/?_website=arc-demo-1
http://localhost/all-blocks/?_website=arc-demo-2
http://localhost/all-blocks/?_website=arc-demo-3
http://localhost/all-blocks/?_website=arc-demo-4


## Effect Of Changes

Text strings are output in the correct language

<img width="223" alt="PEN-1761-1" src="https://user-images.githubusercontent.com/868127/108116127-061d0280-7093-11eb-8fc9-aa83dbbc7e73.png">

<img width="236" alt="PEN-1761-2" src="https://user-images.githubusercontent.com/868127/108116129-06b59900-7093-11eb-95d6-69e2e77b8116.png">

<img width="968" alt="PEN-1761-3" src="https://user-images.githubusercontent.com/868127/108116216-2220a400-7093-11eb-8da5-e4c14e35ca2c.png">


## Dependencies or Side Effects

Requires site settings to be updated in Feature Pack repo - https://github.com/WPMedia/Fusion-News-Theme/pull/157
Requires engine theme SDK PR for date settings - https://github.com/WPMedia/engine-theme-sdk/pull/215


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
